site/app/templates/admin/admin_gradeable/AdminGradeableCreate.twig

```diff
@@ -594,6 +594,11 @@
         //Hide Grade Inquiry Date if Grade Inquiries are not enabled
         onIsRegradeAllowedChange();
         $('[name="regrade_allowed"]').change(onIsRegradeAllowedChange);
+        $('.grade_inquiry_per_component_allowed').change(function() {
+            if ($('#no_grade_inquiry_per_component_allowed').prop('checked') && !confirm('* WARNING *\nThere are already grade inquiries, continuing with this action will permanently convert all component inquiries to general inquiries.  Do you want to proceed?')) {
+                $('#yes_grade_inquiry_per_component_allowed').prop('checked', true);
+            }
+        });
```

Comment:
There is a bug with this. If you click cancel then refresh it still changes the radio button to no.